pages/index.tsx

```diff
@@ -9,7 +9,7 @@ import {TestList} from "../components/CandidateTestView/Tests/TestList";
 export const testToRender: CandidateTestModel = TestList[1];
 const Home: NextPage = () =>
     <Layout>
-        <section className={scss.content}>
+        <body className={scss.content}>
```

Comment:
similarly, I think `section` was right here. Don't think we want the duplicate `body`s.